python/ctsm/test/test_unit_args_utils.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+"""
+Unit tests for arg_utils.py function and types.
+
+You can run this by:
+    python -m unittest test_unit_args_utils.py
+"""
+
+import os
+import sys
+import unittest
+import argparse
+
+# -- add python/ctsm  to path (needed if we want to run the test stand-alone)
+_CTSM_PYTHON = os.path.join(
+            os.path.dirname(os.path.realpath(__file__)), os.pardir, os.pardir)
+sys.path.insert(1, _CTSM_PYTHON)
+
+#pylint: disable=wrong-import-position
+from ctsm.args_utils import plon_type, plat_type
+from ctsm import unit_testing
+
+# pylint: disable=invalid-name
+
+class TestArgsPlon(unittest.TestCase):
+    """
+    Tests for plot_type in args_util.py
+    """
+
```

Comment:
- [ ] I suggest adding checks for exact 90, -90 latitude, -180, 0, 360 longitudes. Do we allow 360 longitude or should that be zero? Also do we allow for exact north and south pole? I find it hard to imagine that would actually be done. But, maybe somebody would want to test there?

A general unit testing thing is to check such exact points where the code says behavior changes. For completeness unit tests should test every line of code and all conditions. Practically you don't always need to do that, but that is the end goal.

Reply:
Thanks @ekluzek for bring this up, I agree that adding test cases for these edge cases are important. 

I added tests for:
- plat_type = -90
- plat_type = 90

- plon_type = -180
- plon_type = 0
- plon_type = 360

Thanks so much for your comments and it is all implemented.